ui/src/main/webapp/src/app/registered-application/register-application-form.component.html

```diff
@@ -1,114 +1,90 @@
 <div class="container-fluid wu-frame">
-    <h1 i18n="Add Applications to Project">Add Applications</h1>
+<h1 i18n="Add Applications to Project">Add Applications</h1>
 
-    <!-- Tabs -->
-    <ul id="addAppModeTabs" class="nav nav-tabs">
-        <li role="presentation" class="active">
-            <a role="tab" data-toggle="tab" aria-controls="tabContentUpload" href="#tabContentUpload" i18n-title="Upload Tab Title" i18n="Upload Tab Text"
-                title="Upload an Application">Upload</a>
-        </li>
-        <li role="presentation">
-            <a role="tab" data-toggle="tab" aria-controls="tabContentRegister" href="#tabContentRegister" i18n-title="Register Server Path Tab Title"
-                i18n="Register Server Path Tab Text" title="Register application file or directory located on the server">Server Path</a>
-        </li>
-    </ul>
+<form [formGroup]="registrationForm" (ngSubmit)="register()"  class="form-horizontal" ng2FileDrop [uploader]="multipartUploader">
 
-
-    <form [formGroup]="registrationForm" (ngSubmit)="register()" class="form-horizontal" ng2FileDrop [uploader]="multipartUploader">
-
-        <div id="addAppsModeTabsContent" class="tab-content">
-
-            <!-- Upload tab panel -->
-            <div role="tabpanel" id="tabContentUpload" class="tab-pane fade in active">
-
-                <div class="row description">
-                    <div class="col-md-10">
-                        <div class="alert" i18n="Upload Description Text">
-                            Choose files to upload to your project. You can select more than one file at a time. You can also drag and drop files to
-                            start uploading.
-                        </div>
+    <wu-tab-container (tabSelected)="onTabSelected($event)">
+        <wu-tab [tabTitle]="'Upload'" [properties]="{mode: 'UPLOADED'}">
+            <div class="row description">
+                <div class="col-md-10">
```

Comment:
I think it depends on what you are trying to create, grid layout with only one item is pointless with col-* styles IMO. If there are more widgets on a line it might work and require to have more specific styles to be responsive and keep the layout.